tests/test_connection_class.py

```diff
@@ -92,6 +92,16 @@ def test_open_twice(self):
         # connection should now be closed
         self.assertFalse(self.plc.is_open)
 
+    def test_netid_port(self):
+        self.assertEqual(self.plc.ams_netid, TEST_SERVER_AMS_NET_ID)
+        self.assertEqual(self.plc.ams_port, TEST_SERVER_AMS_PORT)
+        with self.assertRaises(ValueError):
+            self.plc.ams_netid = "1.1.1.1.1.1.1"
+        self.plc.ams_netid = "1.1.1.1.1.1"
+        self.assertEqual(self.plc.ams_netid, "1.1.1.1.1.1")
+        self.plc.ams_port = 1
+        self.assertEqual(self.plc.ams_port, 1)
+
```

Comment:
Please add a testcase where you write net-id and port of a connection that is already open. An attribute error should be raised.